src/Crucial/Service/Chargify/products-family

```diff
@@ -0,0 +1,83 @@
+<?php namespace Crucial\Service\Chargify;
+
+class ProductFamily extends AbstractEntity
+{
+    /**
+     * List all products for your site
+     *
+     * @return Products by family id
+     */
+    public function listProducts($id)
+    {
+        $service = $this->getService();
+
+        $response      = $service->request('product_families/'.$id.'/products', 'GET');
+        $responseArray = $this->getResponseArray($response);
+
+        if (!$this->isError()) {
+            $this->_data = $this->_normalizeResponseArray($responseArray, 'product');
+        } else {
+            $this->_data = array();
+        }
+
+        return $this;
+    }
+
+    /**
+     * List all products for your site
+     *
+     * @return Family and Products
+     */
+    public function listProductFamilies()
+    {
+        $service = $this->getService();
+
+        $response      = $service->request('product_families', 'GET');
+        $responseArray = $this->getResponseArray($response);
+
+        if (!$this->isError()) {
+            $this->_data = $this->_normalizeResponseArray($responseArray, 'product_family');
+        } else {
+            $this->_data = array();
+        }
+
+        return $this;
+    }
+
+    /**
+     * List all products for your site
```

Comment:
Please update this comment to reflect that it retrieving a product family, not a product.